OpenRA.Game/ObjectCreator.cs

```diff
@@ -54,7 +56,18 @@ public ObjectCreator(Manifest manifest, FileSystem.FileSystem modFiles)
 				Assembly assembly;
 				if (!ResolvedAssemblies.TryGetValue(hash, out assembly))
 				{
-					assembly = Assembly.Load(data);
+					Stream symbolStream = null;
+					var hasSymbols = false;
+
+					// Mono has its own symbol format.
+					if (isMonoRuntime)
+						hasSymbols = modFiles.TryOpen(path + ".mdb", out symbolStream);
+
+					// .NET and newer mono versions can load portable .pdb files.
+					if (!hasSymbols)
```

Comment:
This doesn't seem to work on mono, despite what the release notes lead me to believe.  Can you please change this to an `else`?